src/app/routes/_wiki.router.js

```diff
@@ -0,0 +1,235 @@
+import Wiki from '../models/wiki.model.js';
+import youtubeHelper from '../helper/youtube.js';
+
+export default (app, router, io, admin) => {
+
+  /**
+   * Retrieve the Youtube-Video-ID from a URL
+   * @param  {string} url Link to a Youtube-Video
+   * @return {string}     Video-ID of the video (if url type supported by pattern)
+   */
+  function parseVideoID(url) {
+    // source:
+    // stackoverflow.com/questions/3452546/javascript-regex-how-to-get-youtube-video-id-from-url
+    const regExp = /^.*(youtu\.be\/|v\/|u\/\w\/|embed\/|watch\?v=|\&v=)([^#\&\?]*).*/;
```

Comment:
It should support all sorts of link types. The regexp `youtube\.be` is optional there.
See: http://stackoverflow.com/a/8260383